lib/web_ui/lib/src/engine/text_editing/composition_aware_mixin.dart

```diff
@@ -75,7 +75,7 @@ mixin CompositionAwareMixin {
       return editingState;
```

Comment:
Since we're using `editingState.extentOffset!` for the `composingBase` calculation, we should probably add it to this null check (or replace `editingState.baseOffset` )